stable/kommander/requirements.lock

```diff
@@ -4,12 +4,12 @@ dependencies:
   version: 0.1.6
 - name: kommander-cluster-lifecycle
   repository: https://mesosphere.github.io/kommander-cluster-lifecycle/charts
-  version: 0.5.3
+  version: v0.5.4
```

Comment:
I can't tell whether that's valid or not, just pointing out that you added a `v` in front here.

Reply:
I didn't, helm did it `v2.16.5`

Reply:
this is the result of `helm dep update`